apps/src/code-studio/pd/application/teacher2021/Section4ProfessionalLearningProgramRequirements.jsx

```diff
@@ -274,11 +284,13 @@ export default class Section4SummerWorkshop extends LabeledFormComponent {
               this.renderAssignedWorkshopList()}
           </div>
           {this.radioButtonsFor('willingToTravel')}
-          We offer a virtual academic year workshop track for those who are
+          We offer a series of virtual academic year workshops for those who are
           unable to commit to traveling to in-person academic year workshops.
-          Please note that this option is only available for the academic year -
-          all participants in the Professional Learning Program must attend an
-          in-person five-day summer workshop.
+          This workshop series consists of eight, 1.5 hour live sessions
+          throughout the academic year, hosted via a video conference tool.{' '}
+          <strong>Please note</strong> that this option is only available for
+          the academic year - all participants in the Professional Learning
+          Program must attend an in-person five-day summer workshop.
           {this.radioButtonsFor('interestedInOnlineProgram')}
```

Comment:
Just a content nit, but I find it difficult to parse when a statement is given, and then an exception is presented below.  It might be easier to parse if we broke this into two mutually-exclusive statements, one for academic year, and one for PLP.

Reply:
@agealy ^ thoughts?

Reply:
Hmm, I don't read this as a statement with an exception. It could stand alone as "We offer a series of virtual academic year workshops for those who are unable to commit to traveling to in-person academic year workshops. This workshop series consists of eight 1.5 hour live sessions throughout the academic year hosted via a video conference tool...Do you want to be considered for the virtual academic year workshop track?" but we include an extra sentence in there emphasizing that we are only talking about AYW, not summer. The professional learning program is not separate from AYW, it's the overarching superset

Reply:
Oh I see.  That makes sense.  Thanks.

Reply:
@breville I don't want to dismiss your comment though. I'm probably closer to the details of our programs and how they fit together than the users of this app; there could definitely still be room to improve clarity